SQL/2016-04-02-GenomicBrowser-FileUploader.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE `genomic_analysis_modality_enum` (
+  `analysis_modality` varchar(100),
+  PRIMARY KEY (`analysis_modality`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8
+COMMENT '';
+
+INSERT IGNORE INTO `genomic_analysis_modality_enum` (analysis_modality) VALUES
+('Methylation beta-values'),
+('Other');
```

Comment:
Let's add SNP, CNV and GWAS in here, if it makes sense.  To have ready options pre-populated, and also to suggest to developers what this is for.